### PR TITLE
[INTEROP-7285] Update EAP tests with `pit-7.4.x` branch/tag.

### PR DESCRIPTION
**Pull request by dfrazzette**
Move EAP test config to `pit-7.4.x` config; update periodic pit-7.4.x` jobs; remove deprecated 7.4.x` periodic jobs.

### CODE REVIEW

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-xp-interop-aws

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-interop-aws

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 1]
```diff
@@ -1,4 +1,92 @@
...
+    org: jboss-eap-qe
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.15-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-interop-aws
```
Comment (calebevans) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:19]:
```suggestion
  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-interop-aws
  reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'
```

I know this is still in draft, but it's easy to forget to add this.

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 2]
```diff
@@ -98,6 +186,94 @@ periodics:
...
+    org: jboss-eap-qe
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.16-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.16-lp-interop-eap-74-interop-aws
```
Comment (calebevans) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:206]:
```suggestion
  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.16-lp-interop-eap-74-interop-aws
  reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'
```

**Review by calebevans — COMMENTED** [review]
Looks like you're missing the Slack config (very easy to miss). Everything else looks perfect!

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**